### ISSUE TITLE
Scan to https://bitcoin.org

### ISSUE THREAD

**Issue by Geremia** [leo-bogert/ultimate-vectorized-bitcoin-qr-sticker#1]
Suggestion: Make the QR code should scan to http**s**://bitcoin.org instead.

**Comment by leo-bogert** [comment]
Thanks. I have in fact considered this and would have loved to do so - BUT:
I don't know what the original QR code was generated with!
So to reproduce the sticker in the same style but with a higher quality I had to manually vectorize the image, i.e. recreate it with an image editor, not with something which actually understands how to make a QR code.
Back then I had tried many tools for creating QR codes to get a similar one but IMHO they were all ugly, so I went down this path.

**TL;DR**: I can only do this if someone tells my which tool the QR code was originally generated with.

**Comment by Mushyg1978** [comment]
DailyBTC

**Comment by Mushyg1978** [comment]
https://drive.google.com/open?id=1bKw5dxgD13nohvin4BYD-BD-Bn_vUmnl
